openadapt/scripts/scrub.py

```diff
@@ -19,23 +19,23 @@
     $ python -m openadapt.scripts.scrub scrub_mp4 sample2.mp4 True 2 2 16
 """
 
-from typing import Optional
+from typing import Optional, Any
 import math
 
 from loguru import logger
 from PIL import Image
 from moviepy.editor import VideoFileClip, VideoClip
-from moviepy.video.fx import speedx
 from tqdm import tqdm
 import fire
 import numpy as np
 
 from openadapt import config, scrub, utils
 
 
-def _make_frame(time, final, progress_bar, progress_threshold):
-    """
-    Private function to scrub a frame.
+def _make_frame(
+    time: Any, final: Any, progress_bar: Any, progress_threshold: Any
```

Comment:
```suggestion
    time: int, final: moviepy.video.io.VideoFileClip.VideoFileClip, progress_bar: tqdm.std.tqdm, progress_threshold: int
```

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/d88767ff785961d7eb36ca589d96f707a019b507